packages/wallets/src/components/ErrorState/ErrorState.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import IcCashierError from '../../public/images/ic-cashier-error.svg';
+import FundsProtection from '../FundsProtection';
+import { GenericPageWithAction } from '../GenericPageWithAction';
+
+type TProps = {
+    code: string;
+    message: string;
+};
+
+const ErrorState: React.FC<TProps> = ({
+    code = 'UNKNOWN',
+    message = 'Sorry an error occurred. Please try accessing our cashier page again.',
+}) => {
+    if (code === 'ASK_UK_FUNDS_PROTECTION') return <FundsProtection />;
```

Comment:
minor suggestion for next PR's:

ASK_UK_FUNDS_PROTECTION code is not generic ErrorState, its something specific to WalletDepositFiat.tsx, so probably WalletDepostiFiat.tsx should be responsible for deciding that different error sceen should be displayed.

Reply:
@lubega-deriv  are we going to use this ErrorState in all wallet project to show the errors? and for the second question, are we going to have more of these error codes in the future?

Reply:
@shayan-deriv this needs to be discussed further, will get back to you on it

Reply:
@shayan-deriv for now this line of code will be removed:
`if (code === 'ASK_UK_FUNDS_PROTECTION') return <FundsProtection />;`
as the check would be done in other places hence the code prop would be unneeded.

As for the ErrorState, it would be used as a reusable component to handle various error catch implementations, but for now we'll just be using the current general error message.

cc: @george-usynin-deriv